cloud_service/upgrading-cloud/upgrade-cloudsvc-helm.adoc

```diff
@@ -0,0 +1,25 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="upgrade-cloudsvc-helm"]
+= Upgrading secured clusters in RHACS Cloud Service by using Helm charts
+include::modules/common-attributes.adoc[]
+:context: upgrade-cloudsvc-helm
+
+toc::[]
+
+[role="_abstract"]
+You can upgrade your secured clusters in {product-title-managed-short} by using Helm charts.
+
+If you have installed {product-title-short} secured clusters by using Helm charts, to upgrade to the latest version of {product-title-short} perform the following steps:
+
+. Update the Helm chart.
+. Run the `helm upgrade` command.
+
+[role="_additional-resources"]
```

Comment:
```suggestion

```
In the preview, it looks like maybe the "Updating the Helm chart repository" module is mixed up with an "Additional resources" section.